testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -13,16 +13,72 @@
  */
 package io.trino.testing;
 
+import java.util.function.Predicate;
+
+import static java.util.Objects.requireNonNull;
+
 public enum TestingConnectorBehavior
 {
     SUPPORTS_PREDICATE_PUSHDOWN,
-    SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY,
-    SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY,
+    SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_EQUALITY(SUPPORTS_PREDICATE_PUSHDOWN),
+    SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY(SUPPORTS_PREDICATE_PUSHDOWN),
+
     SUPPORTS_LIMIT_PUSHDOWN,
+
     SUPPORTS_TOPN_PUSHDOWN,
+
     SUPPORTS_AGGREGATION_PUSHDOWN,
-    SUPPORTS_JOIN_PUSHDOWN,
-    SUPPORTS_JOIN_PUSHDOWN_WITH_FULL_JOIN,
-    SUPPORTS_JOIN_PUSHDOWN_WITH_DISTINCT_FROM,
+
+    SUPPORTS_JOIN_PUSHDOWN(
+            // Currently no connector supports Join pushdown by default. JDBC connectors may support Join pushdown and BaseJdbcConnectorTest
+            // verifies truthfulness of SUPPORTS_JOIN_PUSHDOWN declaration, so it is a safe default.
+            false),
+    SUPPORTS_JOIN_PUSHDOWN_WITH_FULL_JOIN(SUPPORTS_JOIN_PUSHDOWN),
+    SUPPORTS_JOIN_PUSHDOWN_WITH_DISTINCT_FROM(SUPPORTS_JOIN_PUSHDOWN),
+
+    SUPPORTS_CREATE_TABLE,
+    SUPPORTS_CREATE_TABLE_WITH_DATA(SUPPORTS_CREATE_TABLE),
```

Comment:
Why not `SUPPORTS_CREATE_TABLE_AS`?

Reply:
"AS" is too short, "WITH DATA" may be confusing... any other options?

Reply:
`SUPPORTS_CREATE_TABLE_AS_SELECT` is probably better than just `_AS`.

Though I think `WITH_DATA` is probably fine. Maybe it could have a doc comment to help anyone who gets confused (some of the others would probably also benefit from doc comments, like `JOIN_PUSHDOWN_WITH_DISTINCT_FROM`).

(A doc comment for this one could just be `{@code CREATE TABLE name AS query}`.)

Reply:
yes, we can add comments here!